.pre-commit-config.yaml

```diff
@@ -0,0 +1,55 @@
+ci:
+  autofix_prs: false
+
+repos:
+  # Lint first (with safe autofixes)
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.6.9
+    hooks:
+      - id: ruff
+        name: ruff-check
+        args: [--fix, --show-fixes]
+        files: ^(kubeflow-pipelines/(docling-standard|docling-vlm)/.*\.py|[^/]+\.py)$
+
+  # Fast formatter (optional alongside Black; keeps layout consistent)
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.6.9
```

Comment:
This version of `ruff-pre-commit` is very older too. Newer versions are `v0.13`. I understanding not wanting to be on the bleeding edge but any reason we should be on `v0.6.9`?